tests/integration/components/frost-error-test.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * Integration test for the frost-error component
+ */
+
+import {expect} from 'chai'
+import {$hook} from 'ember-hook'
+import {integration} from 'ember-test-utils/test-support/setup-component-test'
+import hbs from 'htmlbars-inline-precompile'
+import {beforeEach, describe, it} from 'mocha'
+
+const test = integration('frost-error')
+describe(test.label, function () {
+  test.setup()
+  describe('when hook is passed in', function () {
+    beforeEach(function () {
+      this.render(hbs`
+        {{frost-error
+          hook="fe"
+        }}
+      `)
+    })
+
+    it('should add hook to description', function () {
+      expect($hook('fe-description')).to.have.class('frost-error-description')
```

Comment:
I'd suggest using variables for the classes, find/replace isn't terrible, but a prefix variable is better